Source/Core/Core/USBUtils.cpp

```diff
@@ -0,0 +1,294 @@
+// Copyright 2017 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/USBUtils.h"
+
+#include <algorithm>
+#include <charconv>
+#include <cwchar>
+#include <functional>
+#include <optional>
+#include <ranges>
```

Comment:
`<ranges>` is unused. `std::ranges::find_if` apparently comes from `<algorithm>`.